src/fenics_constitutive/models/utils.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import annotations
+
+
```

Comment:
Since there is only one function in this file, I would move that to the materialtensor.py instead.
I think, there will not be a lot of further utility functions in the future